torchvision/ops/boxes.py

```diff
@@ -6,7 +6,16 @@
 from torchvision.extension import _assert_has_ops
 
 from ..utils import _log_api_usage_once
-from ._box_convert import _box_cxcywh_to_xyxy, _box_xywh_to_xyxy, _box_xyxy_to_cxcywh, _box_xyxy_to_xywh
+from ._box_convert import (
+    _box_cxcywh_to_xyxy,
+    _box_cxcywhr_to_xywhr,
+    _box_cxcywhr_to_xyxyxyxy,
+    _box_xywh_to_xyxy,
+    _box_xywhr_to_cxcywhr,
+    _box_xyxy_to_cxcywh,
+    _box_xyxy_to_xywh,
+    _box_xyxyxyxy_to_cxcywhr,
+)
```

Comment:
I originally thought we should try to only have one single implementation of the conversion logic, first within the transforms.

But I understand now that there is value in having the complex implementation in `transforms` that supports the `inplace` logic, and a more trivial implementation here that can serve as implementation reference, which we rely on for the tests.

So, I agree it's OK to expose these here within `box_convert`. Thank you!